### DISCUSSION TITLE
Change the default Dashboard

### DISCUSSION THREAD

**Discussion by KevinFarrow** [abpframework/abp#6285] [Q&A] [answered]
How do we change the default Dashboard? or the Tenant Dashboard? in the Teams version of abp commercial? Can we add our own widgets and change the layout? or would we replace the page completely?

**Comment by yekalkan** [comment, marked as answer]
> How do we change the default Dashboard? or the Tenant Dashboard? in the Teams version of abp commercial?

Dashboard pages don't belong to any module and are in final application template. That means you will have the source code of these pages whatever your license plan is. However, each widget has it's own view component and they are in module source code.

> Can we add our own widgets and change the layout?  or would we replace the page completely?

Yes.

**Comment by KevinFarrow** [comment]
Thanks for your quick reply @yekalkan - much appreciated.
